components/admin_console/plugin_management/plugin_management.jsx

```diff
@@ -1068,4 +1069,4 @@ export default class PluginManagement extends AdminSettings {
             </div>
         );
     }
-}
+}
```

Comment:
nit: please revert.  Convention is with newline as eol.  Same to other instances.

Reply:
Thanks @saturninoabril . Updated the comment